TypeCobol.Test/Parser/Programs/Cobol85/GlobalNested.cbl

```diff
@@ -0,0 +1,43 @@
+IDENTIFICATION DIVISION.
+PROGRAM-ID. EnclosingPgm.
+DATA DIVISION.
+LOCAL-STORAGE SECTION.
+
+01 MyGlobalVar PIC X(5) GLOBAL.
+01 MyNotGlobalVar PIC X(5).
+
+01 MyGlobalGroup GLOBAL.
+  05 MGG1 PIC X(5).
```

Comment:
A data description that contains children cannot have a picture.
Our parser don't check this rule yet.

Can you replace with:
05 MGG1.